submit_workflow_source_files/submit_workflow_source_files.py

```diff
@@ -0,0 +1,100 @@
+from common import *
```

Comment:
My fault, but this folder should be in the scripts folder for consistency.

Reply:
Didn't mean to add that, removed it in updates